tutorials/node/lightning-network-daemon-linux/tutorial.yml

```
@@ -0,0 +1,17 @@
+id: 59d777e9-72c8-4b32-8c50-e86cdae8f2f9
+project_id: db7cc106-432c-4571-a66a-1f68d024bca4
+tags: 
+  - lightning
+  - node
+  - guides
+category : lightning-network
+level : beginner
+professor_id: 81a82a0f-3003-4459-9d6a-67f860efb20e
+original_language: fr
+proofreading: 
+  - language : fr
+    last_contribution:
```

```suggestion
    last_contribution: 2025-04-23
```
Here you should always include the date of the day when you're writing your tutorial